pkg/start/start.go

```diff
@@ -231,21 +231,24 @@ type ClientBuilder struct {
 	config *rest.Config
 }
 
-func (cb *ClientBuilder) RestConfig() *rest.Config {
+func (cb *ClientBuilder) RestConfig(fns ...func(*rest.Config)) *rest.Config {
```

Comment:
```console
$ godoc ./pkg/start
PACKAGE DOCUMENTATION

package start
    import "./pkg/start"


TYPES

type ClientBuilder struct {
    // contains filtered or unexported fields
}


func (cb *ClientBuilder) ClientOrDie(name string, fns ...func(*rest.Config)) clientset.Interface

func (cb *ClientBuilder) KubeClientOrDie(name string, fns ...func(*rest.Config)) kubernetes.Interface

func (cb *ClientBuilder) RestConfig(fns ...func(*rest.Config)) *rest.Config
```

`fns ...` isn't the best `godoc`, maybe either add comments / name the variable that it is evident that these are config modifiers...

Reply:
oh, sorry